lib/Widget/MastodonProvider.php

```diff
@@ -157,4 +170,45 @@ public function getDataModifiedDt(DataProviderInterface $dataProvider): ?Carbon
     {
         return null;
     }
+
+    /**
+     * Get Mastodon Account Id from username
+     * @throws GuzzleException
+     */
+    private function getAccountId(mixed $uri, DataProviderInterface $dataProvider)
```

Comment:
Can we make this function declaration clearer? isn't it `string $uri, string $userName` ?